lib/src/cbor/cbor_initial_byte.c

```diff
@@ -0,0 +1,175 @@
+/**
+ * \file cbor_initial_byte.c
+ *
+ * \author angel.castillo
+ * \date   Sep 12, 2023
+ *
+ * \section LICENSE
+ *
+ * Copyright 2023 Biglup Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "./cbor_initial_byte.h"
+
+#include <cardano/object.h>
+
+#include <stdlib.h>
+
+#include "../config.h"
+
+/* STRUCTS *******************************************************************/
+
+/**
+ * Represents a CBOR initial byte object.
+ */
+typedef struct cardano_cbor_initial_byte_t
+{
+    cardano_object_t base;
+    byte_t           initial_byte;
+    size_t           ref_count;
```

Comment:
ref_count no longer need, additionally this type probably doesn't need to be an object as is just a simple value

Reply:
done here https://github.com/Biglup/cardano-c/pull/6/commits/2a028ae4fe3698180765c52a7a64610944983385